app/models/user.rb

```diff
@@ -48,6 +48,8 @@ class User < ApplicationRecord
            dependent: :destroy
   has_many :submission_documents, class_name: 'Submissions::Document', dependent: :destroy
   has_many :message_context_statuses, dependent: :destroy
+  has_many :challenge_phases_evaluators, dependent: :destroy
+  has_many :evaluated_challenges, through: :challenge_phases_evaluators, source: :challenge
```

Comment:
this is a valid relationship but I think the phases might be more useful since they will be unique while there will be duplicate challenges per user